### PR TITLE
Interfaces updated

### PR DESCRIPTION
**Pull request by sergiishamrai-okta**
OktaSignOnPolicyConditions should implement PolicyRuleConditions
PasswordPolicyConditions should implement PolicyRuleConditions

### CODE REVIEW

File: dist/spec.json [hunk 1]
```diff
@@ -15771,6 +15771,7 @@
           "$ref": "#/definitions/PolicyPeopleCondition"
         }
       },
+      "x-okta-parent": "#/definition/PolicyRuleConditions",
```
Comment (serhiibuniak-okta) [inline dist/spec.json:15774]: probably, should be spelled definition with 's' at the end: definitions

File: dist/spec.json [hunk 2]
```diff
@@ -16204,6 +16205,7 @@
...
```
Comment (serhiibuniak-okta) [inline dist/spec.json:16208]: the same as above and rest of the changes

File: dist/spec.json (hunk 1 above)
Comment (sergiishamrai-okta) [reply dist/spec.json]: yes, thanks

**Review by serhiibuniak-okta — APPROVED** [review]

**Review by andriizhegurov-okta — APPROVED** [review]
👍

**Review by bretterer — APPROVED** [review]

**Pull request merged**